### ISSUE TITLE
Question: can I define styles that are dynamically applied

### ISSUE THREAD

**Issue by Zhami** [morishitter/picostyle#29]
I'd like to decorate some components based upon prop values passed in to them.
Can I use picostyle to define a class which I can then add or remove when computing the component?
Or is there some other way to use picostyle to handle dynamic styling.

**Comment by B-Teague** [comment]
Yes, you can conditionally assign classes based on a state variable.

Const isError = false;
const redClass = css({
   color: "red"
})
const greenClass = css({
   color: "green"
})

div class={isError ? redClass : greenClass}
